travel_platform/Html/main.html

```diff
@@ -0,0 +1,100 @@
+{% extends 'blog/base.html' %}
+{% block content %}
+<header>
+	<div class="row head-bkg">
+		<div class="col-md-offset-3 col-md-6 head-text text-center">YOUR<br>TRAVEL</div>
+	</div>
+	</header>
+
+	<div class="row">
+	<div class="col-md-offset-4 col-md-8"><h2 class="text-center white">Common information</h2></div>
+	<div class="col-md-4 choose"><a href="https://lviv.itea.ua/">
```

Comment:
it's difficult to understand which 'div' is closed and which is not. please add additional spaces for that.